packages/solid-query/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "author": "tannerlinsley",
+  "dependencies": {
+    "@tanstack/query-core": "4.3.9"
```

Comment:
hmm not sure where 4.3.9 is coming from, as the latest release is 4.3.8 ? I think this makes the codesandbox preview build fail!

Reply:
This was my fault, I thought all versions were pinned together, so I took the version from react-query and got a bit too excited, and submitted the PR with bugs.

Reply:
This should have been fixed in the latest changes